.eslintrc.json

```diff
@@ -8,7 +8,6 @@
 
     "parserOptions": {
         "ecmaFeatures": {
-            "experimentalObjectRestSpread": true,
```

Comment:
Would it be possible to keep the spread syntax? I've used it a few times. Ok with tossing it if necessary